schema-registry/src/test/scala/net/manub/embeddedkafka/schemaregistry/EmbeddedKafkaWithSchemaRegistryObjectSpec.scala

```diff
@@ -0,0 +1,55 @@
+package net.manub.embeddedkafka.schemaregistry
+
+import net.manub.embeddedkafka.{EmbeddedKafka, EmbeddedKafkaSpecSupport}
+
+class EmbeddedKafkaWithSchemaRegistryObjectSpec
```

Comment:
Could we drop the "WithSchemaRegistry" part since this class is already on a different package?